boards/arm/mimxrt1160_evk/mimxrt1160_evk.dtsi

```diff
@@ -33,6 +34,26 @@
 			pwms = <&flexpwm1_pwm2 0>;
 		};
 	};
+
+
+	power-states {
+
+		idle: set_point_1_wait {
+			/* idle corresponds to set point 1 (wait) for RT1170 */
+			compatible = "zephyr,power-state";
+			power-state-name="runtime-idle";
+			substate-id = <IMX_SPC_SET_POINT_1_WAIT>;
+			min-residency-us = <10>;
+		};
+
+		suspend: set_point_10_stop {
+			/*  suspend corresponds to set point 10 for RT1170 */
+			compatible = "zephyr,power-state";
+			power-state-name="suspend-to-idle";
+			substate-id = <IMX_SPC_SET_POINT_10_SUSPEND>;
```

Comment:
Curious why set point 10?

Reply:
Set point 10 seemed to have a good combination of lower wakeup time and current consumption. The RT1170 [datasheet](https://www.nxp.com/docs/en/data-sheet/IMXRT1170CEC.pdf) states set point 10 has a typical wakeup time of ~6.74ms (pg 35), and disables all clock sources and PLLs except the 32KHz oscillator (which the GPT runs from). The `imx_spc.h` file defines all set point configurations, so we can switch to a different set point if we need to (as can users when building custom boards)

Reply:
Probably useful to have a comment block somewhere in this code to 1) reference the datasheet and 2) give your rational... That would give future folks some insight in the selection process you went through and provide them with guidance to select (or experiment) with different setpoints potentially in an application dts overlay